# Bioinformatics

My undergraduate research experiences have mostly dealt with analysing the algorithm implemented in the software Mutual for Heuristic Pairwise Aligment and trying to reproduce it. This algorithm compares two samples of de-novo mRNA-Seq data, represented as De-Bruijn graphs, to detect a retrive similar transcripts shared amongst the samples from the graph itself, recovering significantly more shared transcripts. All of these research experience have been under the amazing mentorship of [Dr. Humberto Ortiz Zuazaga](http://ccom.uprrp.edu/~humberto/), and his [MegaProbe Lab](https://ccom.uprrp.edu/~humberto/megaprobe/pages/about.html).

Currently at the laboratory, we are working with developing and testing different algorithms for performing a differential expression analysis on the Sea cucumber (Holothuria glaberrima), that can be found in the Puertorrican coast line.

Some of our awesome lab members and collaborators are [Ivan L. Jimenez Ruiz](https://github.com/Nacrokill/), [Kevin Legarreta](https://github.com/Kevinlega), [Louis Gil](https://github.com/LouisGil), and [Claudio Vega](https://github.com/cvega69).

# Biomedical informatics  

I worked in the field of Causal Discovery at the [University of Pittsburgh](https://www.pitt.edu/) and the [Center for Causal Discovery](https://www.ccd.pitt.edu/) (**CCD**) as part of the iBRICK summer research internship program sponsored by the IDI-BD2K, under the mentorship of [Bryan Andrews](http://www.pitt.edu/~bja43/) and [Dr. Greg Cooper](http://www.ccd.pitt.edu/listings/gregory-cooper/), trying to improve Causal Inference with graphical methods. The research focused on exploring the use of both experimental and non-experimental(observational) datasets in the Causal Inference research pipeline algorithms in order to acquire more information from the data and enrich the inference results.

Within the field of Causal Discovery, I also had the opportunity to work with [Dr. Peter Spirtes](https://www.cmu.edu/dietrich/philosophy/people/faculty/spirtes.html) and [Dr. Joseph Ramsey](https://www.cmu.edu/dietrich/philosophy/people/faculty/ramsey.html) in the  [University of Carnegie Mellon](https://www.cmu.edu/) and in conjunction with the **CCD**, in Dealing with missingness and selection Bias utilizing test-wise deletion as opposed to row-wise deletion. This research focused in creating a tool for the simulation of datasets with missing data, especially selection bias interpreted as a type of conditional missingness, and profiling the ability of certain causal discovery algorithms on working with this type of data set when the simple size is relatively small, and comparing the test-wise deletion approach to the row-wise deletion one.

# Quantitaive Biology Workshop at MIT

On January of 2017, I had the opportunity to visit the MIT Campus through the Neuro-ID & IDI-BD2K programs and attend their awesome [Quantitative biology workshop](https://cbmm.mit.edu/diversity/january-workshop-introduction-computational-tools-used-neuroscience-research) at the [The Center for Brains, Minds and Machines (CBMM)](https://cbmm.mit.edu) which has mostly shaped my interest in graduate studies and research topics. 

## Grants and Scholarship awards:
_Thanks to the following grants and scholarship awards I have been able to participate in such events and research experiences._

[_IDI-BD2K - R25 MD010399_](https://idi-bd2k.hpcf.upr.edu/about/)

[_Claude Shannon Scholarship_](http://ccom.uprrp.edu/~sstem/)

### _Useful Links:_

+ The publication for this algorithm is found [here](http://www.rna-seqblog.com/mutual-heuristic-pairwise-alignment-of-de-bruijn-graphs-to-facilitate-simultaneous-transcript-discovery/).  

+ The weekly research I performed at megaprobe lab is detailed [here](https://github.com/Omig12/megaprobe-lab/blob/master/content/israel.md).

+ The contents of the research is located [here](https://github.com/Omig12/megaprobe-lab/tree/master/content/Mutual_Files).

+ The technical report product of this research is found [here](https://ccom.uprrp.edu/~humberto/megaprobe/images/israel-may-2016.pdf).

+ Evidence of iBRIC participation is [here](https://idi-bd2k.hpcf.upr.edu/2017/08/16/nuestros-estudiantes-en-internados-de-verano-2017-4/
).

+ Algorithms currently being developed and tested for differential expression on the Sea Cucumber organism genome can be found in the following repository [Mutual-Pepino Repo](https://github.com/Omig12/Mutual-pepino)

+ The main repository for differential expression on the Sea Cucumber organism genome is [DBGE](https://github.com/Omig12/DBGDE)
  + This responds to a research iniciative at out campus covered in the following [article](https://www.cienciapr.org/en/monthly-story/sea-cucumbers-and-regeneration).
